modules/oadp-api-details.adoc

```diff
@@ -0,0 +1,231 @@
+
+|`podAnnotations`
+|map [ link:https://pkg.go.dev/builtin#string[string] ] link:https://pkg.go.dev/builtin#string[string]
+|Used to add annotations to pods deployed by Operators.
+
+|`podDnsPolicy`
+|link:https://pkg.go.dev/k8s.io/api/core/v1#DNSPolicy[`corev1.DNSPolicy`]
+|Defines the configuration of the DNS of a pod.
+
+|`podDnsConfig`
+|link:https://pkg.go.dev/k8s.io/api/core/v1#PodDNSConfig[`corev1.PodDNSConfig`]
+|Defines the DNS parameters of a pod in addition to those generated from `DNSPolicy`.
+
+|`backupImages`
+|*link:https://pkg.go.dev/builtin#bool[bool]
+|Used to specify whether or not you want to deploy a registry for enabling backup and restore of images.
+
+|`configuration`
+|*link:https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api/v1alpha1#ApplicationConfig[`ApplicationConfig`]
+|Used to define the data protection application's server configuration.
+
+|`features`
+|*link:https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api/v1alpha1#Features[`Features`]
+|Defines the configuration for the DPA to enable the Technology Preview features.
+|===
+
+link:https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api/v1alpha1#DataProtectionApplicationSpec[Complete schema definitions for the OADP API].
+
+.BackupLocation
+[options="header"]
+|===
+|Property|Type|Description
+
+|`name`
+|link:https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#ObjectMeta[`metav1.ObjectMeta`]
+|Location name.
+
+|`velero`
+|*link:https://velero.io/docs/v1.9/api-types/backupstoragelocation/[velero.BackupStorageLocationSpec]
+|Location to store volume snapshots, as described in link:https://github.com/openshift/oadp-operator/blob/076f1495f6de65b0e96392cb6ef7cff66cf7defb/docs/config/bsl_and_vsl.md[Backup Storage Locations and Volume Snapshot Locations Customization].
+|===
+
+link:https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api/v1alpha1#BackupLocation[Complete schema definitions for the type `BackupLocation`].
+
+.SnapshotLocation
+[options="header"]
+|===
+|Property|Type|Description
+
+|`name`
+|link:https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#ObjectMeta[metav1.ObjectMeta]
+|
+
+|`velero`
+|*link:https://velero.io/docs/v1.9/api-types/volumesnapshotlocation/[velero.VolumeSnapshotLocationSpec]
+|Location to store volume snapshots, as described in link:https://github.com/openshift/oadp-operator/blob/master/docs/config/bsl_and_vsl.md[Volume Snapshot Location Specification].
+|===
+
+linkhttps://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api/v1alpha1#SnapshotLocation[Complete schema definitions for the type `SnapshotLocation`].
+
+.ApplicationConfig
+[options="header"]
+|===
+|Property|Type|Description
+
+|`velero`
+|*link:https://pkg.go.dev/github.com/openshift/oadp-operator/api/v1alpha1#VeleroConfig[VeleroConfig]
```

Comment:
```suggestion
|*link:https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api/v1alpha1#VeleroConfig[VeleroConfig]
```
Just for consistency, I would use the url with a specific commit.

Replace all `https://pkg.go.dev/github.com/openshift/oadp-operator/api` with
`https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api`